autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/ui/credential/saving/declines/AutofillDisablingDeclineCounter.kt

```diff
@@ -136,6 +136,6 @@ class AutofillDisablingDeclineCounter @Inject constructor(
     }
 
     companion object {
-        private const val GLOBAL_DECLINE_COUNT_THRESHOLD = 3
+        private const val GLOBAL_DECLINE_COUNT_THRESHOLD = 2
```

Comment:
I take this should not be here

Reply:
Was just needing a rebase as other branches in the stack have changed/merged already, but didn't want to do it while translations were still in flight. Resolved now.